server/handler.go

```diff
@@ -939,54 +939,51 @@ func (h *Handler) IsLeader() bool {
 
 // PackHistoryHotReadRegions get read hot region info in HistoryHotRegion form.
 func (h *Handler) PackHistoryHotReadRegions() (historyHotRegions []core.HistoryHotRegion, err error) {
-	hotReadLeaderRegions := h.GetHotReadRegions().AsLeader
-	historyLeaderHotRegions, err := h.packHotRegions(hotReadLeaderRegions, true, core.HotRegionTypes[0])
-	if err != nil {
+	hotReadRegions := h.GetHotReadRegions()
+	if hotReadRegions == nil {
 		return
 	}
-	hotReadPeerRegions := h.GetHotReadRegions().AsPeer
-	historyPeerHotRegions, err := h.packHotRegions(hotReadPeerRegions, false, core.HotRegionTypes[0])
+	hotReadPeerRegions := hotReadRegions.AsPeer
+	historyPeerHotRegions, err := h.packHotRegions(hotReadPeerRegions, core.ReadType.String())
 	if err != nil {
 		return
 	}
-	historyHotRegions = append(historyHotRegions, historyLeaderHotRegions...)
 	historyHotRegions = append(historyHotRegions, historyPeerHotRegions...)
 	return
 }
 
 // PackHistoryHotWriteRegions get write hot region info in HistoryHotRegion from
 func (h *Handler) PackHistoryHotWriteRegions() (historyHotRegions []core.HistoryHotRegion, err error) {
```

Comment:
PackHistoryHotReadRegions and PackHistoryHotWriteRegions has some same code ,may decrease cyclomatic complexity

Reply:
sovled